package.json

```diff
@@ -105,6 +105,7 @@
     "eslint-plugin-flowtype": "^5.2.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.1.0",
+    "eslint-plugin-jest-formatting": "^2.0.0",
```

Comment:
This change might be the less consensual. Happy to remove it if you don't like it.
The good thing is that it autofixes.